### ISSUE TITLE
Any limitations for converting trained TF models to ONNX?

### ISSUE THREAD

**Issue by lijing0010** [onnx/onnx-tensorflow#219]
Hi, I'm new to ONNX.
I had a pre-trainned TF model and I want to convert it to ONNX format.
I followed the tutorial， while calling "tensorflow_graph_to_onnx_model", I got an error "NotImplementedError: Conv2DBackpropInput is not implemented."
Is it caused by the deconvolution layer in the tf model?

**Comment by fumihwh** [comment]
`Conv2DBackpropInput` is not implemented in frontend now.
You can contribute to onnx-tf by following implementation of other operators or use `ignore_unimplemented`.

**Comment by lijing0010** [comment]
Hi tjingrant,  isn't "Conv2DBackpropInput " caused by conv2d_transpose?

**Comment by tjingrant** [comment]
Indeed, this op seems to be quite important, I'll implement this one within 1 week.

**Comment by Jiaming-Liu** [comment]
@tjingrant Any news for this?

**Comment by tjingrant** [comment]
Sorry for the delay, will look at it today. In the mean time, do you happen to have an example onnx model that requires conv2d_transpose?
